interface-elements-for-web/articles/grid/customization-dialog/grouping-page.md

```diff
@@ -4,10 +4,12 @@ author: Anna Kondratova
 legacyId: 118821
 ---
 # Grouping Page
-Select a column in the "Columns to Group" list to group grid data.  The grid data can be grouped by more than one column, and a list of the grouped columns is displayed at the top of **Grouping** page.
+The **Grouping** page enables you to group grid data and change the order of grouping columns within the grid.
+
+Select a column's check box in the **Columns to Group** list to group grid data against this column.  The grid data can be grouped by more than one column; a list of the grouped columns is displayed at the top of the **Grouping** page.
 * Use ![eud-grid-customizationdialog-sorting-drag](../../../images/img128887.png) to drag the columns in this list and change the grouping priority.
-* Use ![eud-grid-customizationdialog-sorting-delete](../../../images/img128889.png) to remove a column from the list of selected columns.
+* Use ![eud-grid-customizationdialog-sorting-delete](../../../images/img128889.png) or clear a column's check box in the **Columns to Group** list, to remove a column from the grouping list.
```

Comment:
check box  -> checkbox

Reply:
"list, to remove" Why do you use a comma here?

Reply:
Microsoft recommend "[Use check box, not box or checkbox, to refer to a check box in UI.](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/check-check-box-check-mark)"